vterm.el

```diff
@@ -187,10 +197,22 @@ be send to the terminal."
 
 Then triggers a redraw from the module."
   (with-current-buffer (process-buffer process)
-    (vterm--write-input vterm--term input)
-    (let ((inhibit-read-only t)
-          (inhibit-redisplay t))
-      (vterm--update vterm--term))))
+    (vterm--write-input vterm--term input)))
+
+
+(defun vterm--run-timer (buffer)
+  (interactive)
+  (let ((inhibit-redisplay t)
+        (inhibit-read-only t))
+    (when (buffer-live-p buffer)
+      (with-current-buffer buffer
+        (vterm--update vterm--term)))))
+
```

Comment:
Is there any particular reason to change the event-driven I/O to timer-based polling?

Reply:
I think don't need to response to  all I/O  event .

Reply:
My terminals are by and large idle 99% of the time. I think the event-driven updates were fine, and nothing about that design prevents us from ignoring updates on certain events if that is ever a problem.

Reply:
ok the timer is removed